Casks/transmit4.rb

```diff
@@ -2,7 +2,7 @@
   version '4.4.13'
   sha256 '0d7095351378aa3c581de064f99fedecbfac683377048a4c0457976f5ce79f3b'
 
-  url "https://www.panic.com/transmit/d/Transmit%20#{version}.zip"
+  url "https://download.panic.com/transmit/Transmit%204/Transmit%20#{version}.zip"
```

Comment:
```suggestion
  url "https://download.panic.com/transmit/Transmit%20#{version.major}/Transmit%20#{version}.zip"
```

In the future, please keep the option to allow edits from maintainers.

Reply:
This should be fixed now.

Just made little sense to me to support non-version-4 downloads on a cask specific to version 4. (The same goes for the appcast, which uses `#{version.major}` but does not generate a valid URL for any of the versions prior to 4…)

Reply:
Those make the cask a stronger base for others to reference.